web/pdf_page_view.js

```diff
@@ -513,6 +513,22 @@ class PDFPageView {
     }
   }
 
+  /**
+   * @typedef {Object} PDFPageViewUpdateParameters
+   *
```

Comment:
Nit: We usually don't bother with a blank line in these cases
```suggestion
   * @typedef {Object} PDFPageViewUpdateParameters
```